common/rpc/interceptor/rate_limit.go

```diff
@@ -91,6 +91,11 @@ func (i *RateLimitInterceptor) Allow(
 		token = RateLimitDefaultToken
 	}
 
+	// we don't want to apply rate limiter if a method is configured with 0 tokens.
+	if token == 0 {
```

Comment:
We should probably reject when token < 1 /just in case/.
```suggestion
	if token < 1 {
```